extensions/functions_datetime.yaml

```diff
@@ -688,3 +688,29 @@ scalar_functions:
           - name: origin
             value: time
         return: time
+  -
+    name: "current_date"
+    description: >-
+      Return the current date in the local time zone.
+
+      The `mode` option determines how often the date is evaluated. STREAMING mode
+      evaluates the current date for each row. BATCH mode evaluates the current date
+      once when the query starts and uses the same result for every row.
+    impls:
+      - args:
+          - name: mode
+            options: [ STREAMING, BATCH ]
+        return: date
+  -
+    name: "current_timestamp"
+    description: >-
+      Return the current timestamp in the local time zone.
```

Comment:
Why do we need to involve time zones?  The "current timestamp" is an instant in time.  It should be the same regardless of the time zone of the consumer.

Reply:
Yeah, it's not needed.  Just removed it.  Thanks!